src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -487,6 +495,10 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
             if (!controllerData.pick) {
                 return;
             }
+            if (controllerData.screenCoordinates) {

```

Comment:
could we factorize this block ? it looks similar in the 4 cases ?

Reply:
The pointerEvent object is reused on a lot of those, but is not reused between different types of ray modes.

Reply:
let me see if I can find a nice way to unify those.